package.json

```diff
@@ -7,19 +7,17 @@
     "data": "npm run data-airlines && npm run data-aircrafts",
     "data-airlines": "mkdir -p data && curl -o data/airlines.csv https://raw.githubusercontent.com/jpatokal/openflights/master/data/airlines.dat",
     "data-aircrafts": "mkdir -p data && curl -o data/aircrafts.zip https://junzis.com/adb/download && unzip data/aircrafts.zip aircraft_db.csv -d data && mv data/aircraft_db.csv data/aircrafts.csv && rm -rf data/aircrafts.zip",
-    "postinstall": "npx electron-rebuild",
+    "postinstall": "./node_modules/.bin/MagicMirror-rebuild",
```

Comment:
Can't we use `npx magicmirror-rebuild`? `npzx` is there to avoid fiddling with path :)

Reply:
I don't actually have my mirror up and running at the moment to test this, but I can take your word for it and amend if you wish?

Reply:
We can change that latter if we need to, it's fine 👍